tests/aws/aws_kubernetes.rs

```diff
@@ -259,97 +149,97 @@ fn create_eks_cluster_in_eu_west_3() {
-//     let session = engine.session().unwrap();
-//     let mut tx = session.transaction();
-//
-//     let aws = test_utilities::aws::cloud_provider_aws(&context);
-//     let nodes = test_utilities::aws::aws_kubernetes_nodes();
-//
-//     let cloudflare = dns_provider_cloudflare(&context);
-//
-//     let mut file = File::open("tests/assets/eks-options.json").unwrap();
-//     let mut read_buf = String::new();
-//     file.read_to_string(&mut read_buf).unwrap();
-//
-//     let options_result = serde_json::from_str::<
-//         qovery_engine::cloud_provider::aws::kubernetes::Options,
-//     >(read_buf.as_str());
-//
-//     let region = "eu-west-3";
-//     let kubernetes = EKS::new(
-//         context,
-//         generate_cluster_id(region).as_str(),
-//         generate_cluster_id(region).as_str(),
-//         AWS_KUBERNETES_VERSION,
-//         region,
-//         &aws,
-//         &cloudflare,
-//         options_result.expect("Oh my god an error in test... Options options options"),
-//         nodes,
-//     );
-//
-//     match tx.delete_kubernetes(&kubernetes) {
-//         Err(err) => panic!("{:?}", err),
-//         _ => {}
-//     }
-//
-//     let _ = match tx.commit() {
-//         TransactionResult::Ok => assert!(true),
-//         TransactionResult::Rollback(_) => assert!(false),
-//         TransactionResult::UnrecoverableError(_, _) => assert!(false),
-//     };
-// }
+fn delete_eks_cluster_in_us_east_2() {
+     init();
+
+     let context = test_utilities::aws::context();
+
+     let engine = test_utilities::aws::docker_ecr_aws_engine(&context);
+     let session = engine.session().unwrap();
+     let mut tx = session.transaction();
+
+     let aws = test_utilities::aws::cloud_provider_aws(&context);
+     let nodes = test_utilities::aws::aws_kubernetes_nodes();
+
+     let cloudflare = dns_provider_cloudflare(&context);
+
+     let mut file = File::open("tests/assets/eks-options.json").unwrap();
+     let mut read_buf = String::new();
+     file.read_to_string(&mut read_buf).unwrap();
+
+     let options_result = serde_json::from_str::<
+         qovery_engine::cloud_provider::aws::kubernetes::Options,
+     >(read_buf.as_str());
+
+     let region = "us-east-2";
+     let kubernetes = EKS::new(
+         context,
+         generate_cluster_id(region).as_str(),
+         generate_cluster_id(region).as_str(),
+         AWS_KUBERNETES_VERSION,
+         region,
+         &aws,
+         &cloudflare,
+         options_result.expect("Oh my god an error in test... Options options options"),
+         nodes,
+     );
+
+     match tx.delete_kubernetes(&kubernetes) {
+         Err(err) => panic!("{:?}", err),
+         _ => {}
+     }
+
+     let _ = match tx.commit() {
+         TransactionResult::Ok => assert!(true),
+         TransactionResult::Rollback(_) => assert!(false),
+         TransactionResult::UnrecoverableError(_, _) => assert!(false),
+     };
+ }
+
+fn delete_eks_cluster_in_eu_west_3() {
```

Comment:
what if the cluster is not deployed? wouldn't it be less error prone to integrate it directly in the create_and_destroy_eks_cluster_in_eu_west_3 for example?

Reply:
It s not a test